wordpress-mobile/WordPress-iOS
# Strip "+" and "00" from country prefix code before sending it to backend for validation

Fixes #12181. 

To test:
* Go to `RegisterDomainDetailsViewModel.swift`, line 190 and add `return;` there, to prevent you from actually redeeming the domain, and put a breakpoint there
* Follow the steps outlined in #12181 
* Verify that no validation error gets shown and the breakpoint you set gets hit
